kubernetes-manifests/server.deployment.yaml

```diff
@@ -42,6 +42,9 @@ spec:
           ${prestop_hook.main_lifecycle() if pre_stop_hook else ''}
           args:
             - "--port=${test_port}"
+            % if add_dualstack_support:
+            - "--address_type=IPV4"
+            % endif
```

Comment:
This isn't needed, it is better to just leave it to the default which will use the wildcard like it does today.

Reply:
do you mean remove the if block and keep "--address_type=IPV4" as default?

Reply:
No, I mean that we shouldn't specify `address_type` except on tests that specifically want to limit which IP version is used.  When the server sees no `address_type` argument it will use the wildcard address.  I made sure to specifically handle that case so that existing tests would not be affected.  (not relevant, but possibly interesting is that `IPV4_IPV6` is the default value used when no address_type is specified)

Reply:
Done

Reply:
After refreshing I see the changes in the delete method, but still see the added if statement here that should be removed.

Reply:
Removed. Looks like that was overriden too